python/feathub/feature_views/sliding_feature_view.py

```diff
@@ -40,15 +45,18 @@ class SlidingFeatureView(FeatureView):
     change in the source table.
 
     The output fields of this feature view consist of the features explicitly listed in
-    the `features` parameter, their group-by keys, and the timestamp field (if exists)
-    in the source table. Other fields in the source table will not be included.
+    the `features` parameter, their group-by keys, and an additional window timestamp
+    field with default name `window_time` and format "%Y-%m-%d %H:%M:%S.%f". Other
+    fields in the source table will not be included.
     """
 
     def __init__(
         self,
         name: str,
         source: Union[str, TableDescriptor],
         features: Sequence[Union[str, Feature]],
+        timestamp_field: str = "window_time",
+        timestamp_format: str = "%Y-%m-%d %H:%M:%S.%f",
```

Comment:
More existing classes (e.g. `TableDescriptor`) use `epoch` as the default value for `timestamp`. Would it be more consistent and more performant to use the same default value here?

Reply:
`epoch` is not the default value of `timestamp_format`. A TODO is added to support epoch with millisecond precision.